contracts/campaignAirdrops/NewSB1155DropCampaign.sol

```diff
@@ -8,24 +8,20 @@ import "./shared/CampaignAirdropsShared.sol";
 /// @title Airdrops new SoulBound ERC1155 token for airdrop recipients
 contract NewSB1155DropCampaign is ERC1155Upgradeable, CampaignAidropsShared {
     uint256 private constant _tokenId = 0;
-    uint256 private constant _tokenAmount = 1;
-    uint256 public constant tokensPerClaim = 1; // 1 reward per wallet
+    uint256 private constant _tokenAmount = 1; // 1 token per claim
     string public constant airdropType = "SB1155";
 
     string public name;
     string public symbol;
     string public contractURI;
     bool public contractURIset;
```

Comment:
Do we really need this bool?
Looks like it should behave like `initializer` in `setContractURI`, but it isn't,